index.bs

```diff
@@ -0,0 +1,139 @@
+ * inactive: the AudioSession is not playing sound.
+
+The page has a default audio session which is used by the user agent to automatically set up the audio session parameters.
+The UA will request and abandon audio focus when media elements start/finish playing on the page.
+This default audio session is represented as an `AudioSession` object that is exposed as `navigator.audioSession`.
+
+<pre class="idl">
+enum AudioSessionState {
+  "inactive",
+  "active",
+  "interrupted"
+};
+
+enum AudioSessionType {
+  "auto",
+  "playback",
+  "transient",
+  "transient-solo",
+  "ambient",
+  "play-and-record"
+};
+
+[Exposed=Window]
+partial interface Navigator {
+  // The default audio session that the user agent will use when media elements start/stop playing.
+  readonly attribute AudioSession audioSession;
+};
+
+[Exposed=Window]
+interface AudioSession : EventTarget {
+  attribute AudioSessionType type;
+
+  readonly attribute AudioSessionState state;
+  attribute EventHandler onstatechange;
+};
+</pre>
+
+# Privacy considerations # {#privacy}
+
+# Security considerations # {#security}
+
+# Examples # {#examples}
+
+## A site sets its audio session type proactively to "play-and-record" ## {#proactive-play-and-record-example}
+
+```javascript
+navigator.audioSession.type = 'play-and-record';
+// From now on, volume might be set based on 'play-and-record'.
+...
+// Start playing remote media
+remoteVideo.srcObject = remoteMediaStream;
+remoteVideo.play();
+// Start capturing
+navigator.mediaDevices.getUserMedia({ audio:true, video:true }).then(stream => {
+    localVideo.srcObject = stream;
+});
+```
+
+## A site reacts upon interruption ## {#interrutpion-handling-example}
+
+```javascript
+navigator.audioSession.type = 'play-and-record';
+// From now on, volume might be set based on 'play-and-record'.
+...
+// Start playing remote media
+remoteVideo.srcObject = remoteMediaStream;
+remoteVideo.play();
+// Start capturing
+navigator.mediaDevices.getUserMedia({ audio:true, video:true }).then(stream => {
+    localVideo.srcObject = stream;
+});
+let isInterrupted = false;
+navigator.audioSession.onstatechange = () => {
+    if (navigator.audioSession.state === 'interrupted') {
+        isInterrupted = true;
+        localVideo.pause();
+        remoteVideo.pause();
+        // Make it clear to the user that the call is interrupted.
+        showInterruptedBanner();
+        localVideo.srcObject.getTracks().forEach(track => track.enabled = false);
+        return;
+    }
+    if (isInterrupted) {
+        isInterrupted = false;
+        // Let user decide when to restart the call.
+        showOptionalRestartBanner().then((result) => {
+            if (!result)
+                return;
+            localVideo.srcObject.getTracks().forEach(track => track.enabled = true);
+            localVideo.play();
+            remoteVideo.play();
+        });
+    }
+}
+```
```

Comment:
Add a `Acknowledgments` section as well?

Reply:
Sounds good, I'll add a section for now.
I filed https://github.com/w3c/audio-session/issues/15